src/openforms/contrib/zgw/clients/zaken.py

```diff
@@ -22,6 +22,7 @@ def create_zaak(
         vertrouwelijkheidaanduiding: str = "",
         payment_required: bool = False,
         existing_reference: str = "",
+        productenOfDiensten: list[str] = [],
```

Comment:
*never ever* use mutable default arguments in python (like lists and dicts) - if the function/method mutates them, the default argument for the next call is mutated.

instead, use `None` and handle the empty list default (if needed) in the function body.